### PR TITLE
refactor: whitespace and code formatting

### PR DESCRIPTION
**Pull request by bogdan-manole**

### CODE REVIEW

File: contracts/interfaces/for-export/IAedexV2Router.aes [hunk 1]
```diff
@@ -27,7 +27,7 @@ contract interface IAedexV2Pair =
   entrypoint domain_separator : () => bytes(32)
   entrypoint nonces : (address) => int
   entrypoint balance : (address) => option(int)
-  entrypoint init : (IAedexV2FactoryForPair, IAEX9Minimal, IAEX9Minimal, int, option(int)) => void
+  entrypoint init : (IAedexV2FactoryForPair, IAEX9Minimal, IAEX9Minimal, option(int), option(int)) => void
```
Comment (ghallak) [inline contracts/interfaces/for-export/IAedexV2Router.aes:30]: The title of the PR does not mention this change. Should this change be here?

**Review by ghallak — COMMENTED** [review]
Nice attempt for better formatting consistency, but this might need a second look, for example in the file `contracts/AedexV2Pair.aes`, at line 29, a couple of spaces are needed before `fee_to_setter`. In the file `contracts/AedexV2Pair.aes` the lines 25 and 26 are empty. And there are a few more changes that can be done for better formatting consistency.

Beside that and the comment I've added about the change that I'm not sure is related to this PR (changing `int` to `option(int)` in the file `contracts/interfaces/for-export/IAedexV2Router.aes`) all seems good.

File: contracts/interfaces/for-export/IAedexV2Router.aes (hunk 1 above)
Comment (bogdan-manole) [reply contracts/interfaces/for-export/IAedexV2Router.aes]: i excluded the file, is autogenerated, i will make the change in a spearate PR.

**Review by thepiwo — APPROVED** [review]
I also like spaces after `,` (e.g. in arguments, list, tuple) otherwise looks good to me

**Comment by bogdan-manole** [comment]
> I also like spaces after `,` (e.g. in arguments, list, tuple) otherwise looks good to me

i added spaces after all ','

**Pull request merged**